packages/analytics/addon/components/ca-report-preview.hbs

```diff
@@ -4,7 +4,17 @@
       <UkSpinner @ratio={{1}} />
     </div>
   {{else}}
-    <table class="uk-table uk-table-divider uk-table-striped">
+    <UkButton
+      @label={{t "categorySelection.loadMore"}}
+      class="uk-button uk-button-primary uk-float-right uk-margin-small-top"
+      {{on "click" this.exportTable}}
+    >
+      {{t "caluma.analytics.preview.export"}}
+    </UkButton>
```

Comment:
Would probably look better when moved to https://github.com/projectcaluma/ember-caluma/blob/main/packages/analytics/addon/templates/reports/edit.hbs next to the delete button. Moved logic should still work, as the ID is still the same.

Also `@label` needs to be removed as it does nothing, when setting the value in the element.

Reply:
> Would probably look better when moved to https://github.com/projectcaluma/ember-caluma/blob/main/packages/analytics/addon/templates/reports/edit.hbs next to the delete button. Moved logic should still work, as the ID is still the same.

This package exports an HTML-table to an XLSX. The table only exists on the `RESULT TABLE` tab. So the button should only be visible on the tab,

> Also `@label` needs to be removed as it does nothing, when setting the value in the element.

Done.

Reply:
yea seems to looks nicer

Reply:
Make them vertically center aligned at least! My eyes!

Reply:
Also it looks strange to have a regular button next to a icon button. I would make them both regular buttons instead.

Reply:
That's why I just made a quick screenshot and didn't commit anything. I dislike this for 3 reasons (2 of them are already mentioned):

1. vertical alignment (can be fixed)
2. regular button next to icon button
3. most importantly: The delete button refers to the analytics table. The export button refers to the data in the preview. In my feeling they just don't belong together

This is not a hill I want to die on. I can fix 1 & 2 if you think it makes sense.

Reply:
Just looks a bit odd at the top, but we can just let it be.